qcodes/instrument/parameter.py

```diff
@@ -1654,7 +1650,7 @@ def __init__(self, parameter, max_val_age=None):
 
     def get(self):
         """Return latest value if time since get was less than
-        `self.max_val_age`, otherwise perform `get()` and return result
+        `self.max_val_age`, otherwise perform ``get()`` and return result
```

Comment:
i guess the reference to the class attribute should be done properly, right? (but i don't remember how to do it)
```suggestion
        :attr:`max_val_age`, otherwise perform ``get()`` and return result
```

Reply:
I also could not find the proper way to refer to the instance attribute.

using :attr: gives error: 
qcodes.instrument.parameter.GetLatest.get:1: WARNING: py:attr reference target not found: max_val_age

At the moment I have kept as it is. Just removed the self from the attribute.